src/components/inputModal/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { Modal, Form } from "react-bootstrap";
+import React, { useEffect } from "react";
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import FormButtonGroup from "components/inputModal/FormAction";
+import { schema } from "utils/inputFormSchema";
+import { MODAL_TYPE } from "utils/enums";
+import InputTextField from "components/inputModal/InputTextField";
+import InputSelectField from "components/inputModal/InputSelect";
+import {grades, subjects} from "utils/index";
+interface IModalProps {
+  visible: boolean;
+  setVisible: Function;
+  // isEdit: boolean;
+}
+
```

Comment:
There should be a space between imports and the code.

Import should be like: 
1. React Imports
2. UI Library Imports
3. Other Library Imports
4. Local Imports
with Components on top then functions then consts